### ISSUE TITLE
oci-distribution: tag a new release

### ISSUE THREAD

**Issue by flavio** [deislabs/krustlet#616]
Would it be possible to tag a new release of the oci-distribution crate?

A bunch of fixes got merged into the main branch, but they are not yet available inside of the crate. PR like https://github.com/deislabs/krustlet/pull/521 and https://github.com/deislabs/krustlet/pull/593 would be appreciated by the Kubewarden project :smile:

**Comment by thomastaylor312** [comment]
Yeah, we can look into that as soon as we review and merge your latest PR
